### ISSUE TITLE
Is it possible to combine multiple filters into one?

### ISSUE THREAD

**Issue by Bengejd** [peterpeterparker/web-photo-filter#4]
[x] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

I like how this component looks thus far, the only thing I am confused about is it's ability to combine multiple filters into one. 

For example, currently, I am serving filters via CSS, which apply at load-time. One of my filters looks like this:

```
{name: 'Aden', class: 'aden', filter: 'contrast(0.9) brightness(1.2) saturate(0.85) hue-rotate(20deg)'},
```

I noticed that all of the examples that you provide, only allow you to filter based on one value (Hue, brightness, saturation, etc...). So I was curious what goes into creating custom filters, like the one above. 

I checked out the [web-photo-filter-type.tsx](https://github.com/fluster/web-photo-filter/blob/master/src/types/web-photo-filter/web-photo-filter-type.tsx) but I am not sure how to utilize this class to accomplish what I am trying to do, right now. Can you give any insight on the steps required to create a new filter?

**Comment by peterpeterparker** [comment]
@Bengejd first of all, yes you are right, right now it isn't possible to combine multiple filters at once

spontaneously I would say that you need a kind of "matrix combinator", that would give probably a better performance than applying the effect multiple times. something like calling method `getFilter` of `web-photo-filter-type.tsx` for each filters you are interested in and to combine them

pseudo-code:

     const matrix1 = WebPhotoFilterType. getFilter('sepia', null);
     const matrix2 = WebPhotoFilterType. getFilter('greyscale', null);
     const matrix3 = WebPhotoFilterType. getFilter('vintage', null);
     const effectiveMatrix = myCustomMatrixJoin(matrix1, matrix2, matrix3);

**Comment by peterpeterparker** [comment]
Resolved and published in version 3.1.0. Thanks for the idea and sorry for the delay, better now than never isn't 😅
